components/src/atoms/buttons/SecondaryButton.tsx

```diff
@@ -28,23 +28,31 @@ export const SecondaryButton = styled.button.withConfig<SecondaryButtonProps>({
   }
 
   &:hover {
-    opacity: 70%;
+    color: ${props => (props.isDangerous ? COLORS.red60 : COLORS.blue60)};
+    border-color: ${props =>
+      props.isDangerous ? COLORS.red50 : COLORS.blue60};
```

Comment:
from the Figma

![Screenshot 2024-01-24 at 3 02 50 PM](https://github.com/Opentrons/opentrons/assets/474225/7bea607d-39c3-451c-9dab-c72533427935)


```suggestion
      props.isDangerous ? COLORS.red50 : COLORS.blue55};
```

Reply:
oops thank you Koji!